### PR TITLE
Refresh pull request when end is reached

### LINKED ISSUE #79: 'Fetch more PRs' button in the PR list

### PR DESCRIPTION
**Pull request by TimvdLippe**
Fixes #79

### CODE REVIEW

**Comment by eanker** [comment]
Travis and rebase needed to be done

File: src/projects/navigation/project-pull-requests.html [hunk 1]
```diff
@@ -160,11 +160,15 @@
...
+        '_resetSearch(selectedPull)',
+        '_clearTriggers(pullrequests.*)'
       ],
+      attached: function() {
+        this.$.list.setScrollTarget(this);
+      },
       _filter: function(pullrequests, status, searchWord) {
         if (status !== 'All') {
-          pullrequests = pullrequests.filter(function(e) {
+          return pullrequests.filter(function(e) {
```
Comment (TimvdLippe) [inline src/projects/navigation/project-pull-requests.html]: Oops, this is actually incorrect. How did this not fail any tests?!

**Comment by TimvdLippe** [comment]
Tests should be fixed now.

**Comment by txsmith** [comment]
Confirmed not working

**Comment by TimvdLippe** [comment]
Should now work with switching projects.

**Comment by eanker** [comment]
I can confirm that it works.

**Comment by txsmith** [comment]
LGTM!

**Pull request merged**